src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -0,0 +1,33 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet("/cars/drivers/addDrivers")
```

Comment:
```suggestion
@WebServlet("/cars/drivers/add")
```
look into checklist:
Good URLs naming in your project:

/cars/drivers/add - (add driver to car)
/cars/drivers/delete - (remove driver from car)
/cars/drivers/ - (get all drivers from car)
/drivers/add
/drivers/delete
/manufacturers/delete